doc/gs_cluster.md

```diff
@@ -33,7 +33,7 @@ Those commands will install TC8 and optimisation for GS like TurboJPEG, Marlin R
 Then open the */etc/default/tomcat8* file and set **JAVA_OPTS** to : 
 
 ```
--Djava.awt.headless=true -XX:+UseConcMarkSweepGC -Xms2G -Xmx2G -XX:PermSize=256m -DGEOSERVER_DATA_DIR=/opt/geoserver_datadir -Dfile.encoding=UTF8 -Djavax.servlet.request.encoding=UTF-8 -Djavax.servlet.response.encoding=UTF-8 -server -XX:+UseParNewGC -XX:ParallelGCThreads=2 -XX:SoftRefLRUPolicyMSPerMB=36000 -XX:NewRatio=2 -XX:+AggressiveOpts -Djava.library.path=/usr/lib/jni:/opt/libjpeg-turbo/lib64
```

Comment:
GeoServer doc recommends -XX:+UseParallelGC, see http://docs.geoserver.org/stable/en/user/production/container.html but I guess it's outdated

GeoSolutions recommends -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:ParallelGCThreads=4 as per http://geoserver.geo-solutions.it/edu/en/adv_gsconfig/gsproduction.html

Reading http://stackoverflow.com/questions/2101518/difference-between-xxuseparallelgc-and-xxuseparnewgc makes me think that GeoSolutions is right:
```
Apply -XX:+UseParNewGC & -XX:+UseConcMarkSweepGC when you require parallel collection method over YOUNG generation AND require CMS method as your collection over OLD generation memory
```

I'm more concerned by the validity of the other options mentioned here (-XX:SoftRefLRUPolicyMSPerMB=36000 -XX:NewRatio=2 -XX:+AggressiveOpts)